pkg/kcp/nuke/resourceStatusDeleted.go

```diff
@@ -4,14 +4,18 @@ import (
 	"context"
 	cloudcontrolv1beta1 "github.com/kyma-project/cloud-manager/api/cloud-control/v1beta1"
 	"github.com/kyma-project/cloud-manager/pkg/composed"
+	"github.com/kyma-project/cloud-manager/pkg/kcp/nuke/types"
 )
 
 func resourceStatusDeleted(ctx context.Context, st composed.State) (error, context.Context) {
-	state := st.(*State)
+	state := st.(*types.State)
```

Comment:
👍 like here it's obvious it's not State from this package since it's not imported with `.` alias